### DISCUSSION TITLE
How many pages did the 🐶 tear out? 📖

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1230] [Galaxy Trivia] [answered]
A 🐶 was playing with a 📖 and tore out the pages 7, 8, 100, 101, 222 and 223.

**Comment by peterbaccaro** [comment]
**5** pages were torn out.

1. Page 7 and 8 (technically they are the same page in the book)
2. Page 100
3. Page 101
4. Page 222
5. Page 223

**Comment by ghost** [comment]
A single page contains an odd number followed by its successor, which is an even number.

So here, page numbers 7 and 8 are on one single page, then 100 is on a single page, followed by 101 on a single page, 222 on a single page, and finally, 223 on a single page. **_In total, there are 5 pages_**.

**Comment by Better-zyh** [comment, marked as answer]
Assuming page starts with 1 ( 7 and 8 on the same page) : 5 pages in total.
Assuming page starts with 0 (100, 101 in one page and 222, 223 in one page) : 4 pages in total

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/Better-zyh?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. 😊🎉

**Comment by CelioAmaral** [comment]
A 🐶 was playing with a book and tore out the pages 7, 8, 100, 101, 222 and 223. How many pages did he tear off?

Answer is 10 pages.

Obs: 5 leaves were torn out.

A sheet in a book has two pages; an odd page number in front and an even page number behind. Let's take a look at the pages that were cut according to the question;

1. Pages 7 and 8 are on the same sheet so both pages will be torn off (2 pages torn)

2. The next page torn was page 100 whose adjoining page on the same sheet is 99. This means if page 100 was torn, then definitely 99 was also torn (2 more pages torn)

3. The next page torn was 101 whose adjoining page on the same sheet is page 102. If page 101 was torn then page 102 was also definitely torn since they are on the same sheet. (2 more pages torn).

4. The next page cut was page 222 which will be on the same sheet as page 221. So if page 222 was torn then 221 was also torn (2 more pages torn)

5. Finally, page 223 was torn and that meant page 224 was also torn since they’re on the same sheet. (2 more pages torn)

6. To conclude 5 Sheets were torn (bearing pages 7/8, 99/100, 101/102, 221/222, 223/224) and this meant that 10 pages were torn and these are pages 7, 8, 99, 100, 101, 102, 221, 222, 223 and 224.
